src/Core/System/Snippet/SnippetFixer.php

```diff
@@ -15,7 +20,23 @@ public function __construct(private readonly SnippetFileHandler $snippetFileHand
     {
     }
 
-    public function fix(MissingSnippetCollection $missingSnippetCollection): void
+    /**
+     * @deprecated tag:v6.8.0 reason:new-optional-parameter - Will get a second parameter `$invalidPluralization`
+     */
+    public function fix(MissingSnippetCollection $missingSnippetCollection /*, array $invalidPluralization*/): void
+    {
+        /** @var Snippets $invalidPluralization */
+        $invalidPluralization = \func_num_args() === 2 ? func_get_arg(1) : [];
+
+        if (!Feature::isActive('v6.8.0.0') && \func_num_args() < 2) {
+            Feature::triggerDeprecationOrThrow('v6.8.0.0', 'New required parameter `$invalidPluralization` missing');
+        }
```

Comment:
```suggestion
        if (!Feature::isActive('v6.8.0.0')) {
            /** @var Snippets $invalidPluralization */
            $invalidPluralization = \func_num_args() === 2 ? func_get_arg(1) : [];

            if (\func_num_args() < 2) {
                Feature::triggerDeprecationOrThrow('v6.8.0.0', 'New required parameter `$invalidPluralization` missing');
            }
        }
```

this makes it clear, that the whole code can be removed with next major

Reply:
Do I get this wrong? In your case, activating the 6.8 feature flag in 6.7 will lead to an error, since the parameter is commented out, right? That's why I solved it the other way.